packages/appstore/src/components/banners/book-banner/book-banner.scss

```diff
@@ -0,0 +1,46 @@
+.book-banner-template {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1.6rem;
+    margin-block-end: 2.4rem;
+    border-radius: 0.8rem;
+    background: rgba(0, 136, 50, 0.08);
+
+    &__cancel {
+        cursor: pointer;
+    }
+    &__left {
+        display: flex;
+        flex-direction: row;
+    }
+    &__content {
+        display: flex;
+        flex-direction: row;
+        font-size: 1.4rem;
+        line-height: 2.2rem;
+
+        & label {
+            padding-inline: 0.8rem;
+        }
+        & a {
+            font-weight: bold;
+            text-decoration-line: underline;
+            color: $color-black-1;
```

Comment:
@NikitK-deriv does it have the same color in dark them? if not you need to choose the proper color from theme.scss

Reply:
In dark mode it's converting to the white

Reply:
If the color is not consistent in light and dark theme you need to change it to the `var(--text-prominent)`

Reply:
already...